common/lib/xmodule/xmodule/video_module/video_xfields.py

```diff
@@ -173,3 +173,12 @@ class VideoFields(object):
         scope=Scope.settings,
         default="",
     )
+    date_last_view_bumper = DateTime(
+        display_name=_("Date of the last view of the bumper"),
+        scope=Scope.preferences,
+    )
+    do_not_show_again_bumper = Boolean(
```

Comment:
OK. Fixed: «change date_last_view_bumper to bumper_last_view_date."

--
Sincerely,  
Alexander Kryklia
cell: +38 096 722 55 46
skype: auraz_
Отправлено при помощи Sparrow (http://www.sparrowmailapp.com/?sig)

Sent with Sparrow (http://www.sparrowmailapp.com/?sig)  

On Thursday, May 21, 2015 at 14:25, Usman Khalid wrote:

> In common/lib/xmodule/xmodule/video_module/video_xfields.py (https://github.com/edx/edx-platform/pull/7825#discussion_r30793110):
> > @@ -173,3 +173,12 @@ class VideoFields(object): > scope=Scope.settings, > default="", > ) > + date_last_view_bumper = DateTime( > + display_name=_("Date of the last view of the bumper"), > + scope=Scope.preferences, > + ) > + do_not_show_again_bumper = Boolean(  
> @auraz (https://github.com/auraz) Umm, I was thinking more like bumper_show_again would be True by default and once the user clicks "skipsAndDoNotShowAgain" we would set this to False. It was just a small readability suggestion, it is fine as it is. :)
> It would still be nice to change date_last_view_bumper to bumper_last_view_date.
>  
> —
> Reply to this email directly or view it on GitHub (https://github.com/edx/edx-platform/pull/7825/files#r30793110).
>  
>  
>  
>  
>